specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_ApproveRollingUpgrade_MinimumSet_Gen.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
```

Comment:
instead of MinimumSet and MaximumSet examples that look similar, let's have one example with more populated fields.